Library/Formula/docker-machine.rb

```diff
@@ -22,7 +22,9 @@ def install
     mkdir_p buildpath/"src/github.com/docker/"
     ln_sf buildpath, buildpath/"src/github.com/docker/machine"
 
+    ENV["PWD"] = buildpath/"src/github.com/docker/machine/"
```

Comment:
Sounds like it'll need reported and patched upstream to fix this issue; that `PWD` works and `cd` doesn't is pretty much just luck.